lib/display.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Display do
+  def display_board(game_state) do
+    board = %{
+      1 => "1",
+      2 => "2",
+      3 => "3",
+      4 => "4",
+      5 => "5",
+      6 => "6",
+      7 => "7",
+      8 => "8",
+      9 => "9"
+    }
+
+    board = place_markers_on_board(game_state.player_1_mark, game_state.player_1_moves, board)
```

Comment:
Why not just _place_markers_ that takes the _game_state_ any reason you'd ever do only 1 player? Also, why not just keep a board in the _game_state_ and have players return the new state with updates board when the move?

Reply:
This was an artifact from when I was just testing for one player. I think place_markers on board is a great idea. I cannot find a reason to do only one player.